ethash/Cargo.toml

```diff
@@ -29,7 +29,9 @@ bench = []
 [[bench]]
 name = "basic"
 harness = false
+required-features = ["bench"]
 
 [[bench]]
 name = "progpow"
 harness = false
+required-features = ["bench"]
```

Comment:
Looking again, this probably isn't the right thing to do, as it will just skip the benchmark. A compile error might be preferable.

Reply:
Yeah, skipping benchmarks is not what we're trying to achieve here.
We can check ethash benches separately:
```
cargo check --all --benches --exclude ethash
```
and then
```
pushd ethash && cargo check --benches --features=bench && popd
```